crates/next-dev/tests/integration.rs

```diff
@@ -118,26 +118,26 @@ async fn run_test(resource: &str) -> JestRunResult {
         path.to_str().unwrap()
     );
 
-    let test_entry = path.join("index.js");
-    assert!(
-        test_entry.exists(),
-        "Test entry {} must exist.",
-        test_entry.to_str().unwrap()
-    );
+    // Count the number of dirs _under_ crates/next-dev/tests
+    let test_entry = Path::new(resource).join("index.js");
     let package_root = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
     let workspace_root = package_root.parent().unwrap().parent().unwrap();
-    let project_dir = workspace_root.join("crates/next-dev/tests");
+    let project_dir = workspace_root.join(resource);
     let workspace_root = workspace_root.to_string_lossy().to_string();
     let requested_addr = get_free_local_addr().unwrap();
+
     let server = NextDevServerBuilder::new(
         TurboTasks::new(MemoryBackend::new()),
         project_dir.to_string_lossy().to_string(),
         workspace_root,
     )
-    .entry_request("harness.js".into())
-    .entry_request(
-        sys_to_unix(test_entry.strip_prefix("tests").unwrap().to_str().unwrap()).to_string(),
-    )
+    .entry_request(EntryRequest::Module(
+        "@turbo/pack-test-harness".to_string(),
+        "".to_string(),
+    ))
```

Comment:
```suggestion
    .additional_entry_request(EntryRequest::Module(
        "@turbo/pack-test-harness".to_string(),
        "".to_string(),
    ))
```

Perfect would be an option to include some code into all types of modes: CSR, pages and app. But we can do that in a follow-up PR